### ISSUE TITLE
Dots files ignored

### ISSUE THREAD

**Issue by Flavien-Pensato** [prettier/prettier-eslint#107]
**Versions**:

- `prettier-eslint` version: 3.4.2
- `node` version: v7.10.0
- `yarn` version: v0.23.0

**Relevant code or config**

```javascript
/* eslint no-console: "off", prettier/prettier: ["error", {		
  "trailingComma": "es5",		
  "singleQuote": true,		
  "printWidth": 120		
}] */

const _ = require("lodash");
```

**What I did:**

prettier-eslint --write .pygrc.js

**What happened:**

No single quote appear.

**Problem description:**

I know that with eslint we have to use `--ignore-pattern '!.pygrc.js'` because by default he is ignoring dot files.

But with prettier-eslint it's not working.

**Suggested solution:**

We think that it's a bug from prettier-eslint to ignore by default dotfiles.

**Comment by zimme** [comment]
@Flavien-Pensato Can you confirm that this is still an issue with the latest version?

**Comment by zimme** [comment]
Then problem here was that you were using `eslint-plugin-prettier`s `prettier/prettier` rule to configure prettier, which `prettier-eslint` didn't have support for inferring prettier config from.

This is now released in version 8.6.0.

I'm closing this as your example should be working with the latest version.

**Comment by unordered** [comment]
same like [this issue](https://github.com/prettier/prettier-atom/issues/339#issuecomment-429212213)
